src/wp-includes/widgets.php

```diff
@@ -1072,22 +1072,37 @@ function wp_get_sidebar( $id ) {
  * Set the sidebar widget option to update sidebars.
  *
  * @since 2.2.0
+ * @since 6.2.0 Added the optional $refresh_global_sidebars_widgets argument.
```

Comment:
```suggestion
 * @since 6.2.0 Added the optional `$refresh_global_sidebars_widgets` parameter.
```

Update the document format and use parameter instead of argument for consistency.